src/js/control-bar/mute-toggle.js

```diff
@@ -30,13 +30,6 @@ class MuteToggle extends Button {
     });
   }
 
-  createEl() {
-    return super.createEl('div', {
-      className: this.buildCSSClass(),
-      innerHTML: `<div><span class="vjs-control-text">${this.localize('Mute')}</span></div>`
```

Comment:
After removing this, I don't see anything that adds the control text on init. Update may not get called immediately.